lib/BC/Optimizer.cpp

```diff
@@ -56,7 +56,10 @@ void OptimizeModule(const remill::Arch *arch, llvm::Module *module,
   TLI->disableAllFunctions();  // `-fno-builtin`.
 
   llvm::PassManagerBuilder builder;
-  builder.OptLevel = 3;
+  // TODO(alex): Some of the optimization passes that the builder adds still rely on typed pointers
+  // so we cannot use them. We should switch to using the new pass manager and choose which passes
+  // we want.
+  builder.OptLevel = 0;
```

Comment:
This isn't ideal but it looks like Anvill doesn't actually use this. It's just `remill-lift` that calls into it so I'm assuming the impact of this is minimal.